std/equal.m31

```diff
@@ -375,6 +375,16 @@ let process_eta =
     | |- ?e : ?t => deep [] e t
     end
 
+let process_reducing c l =
+  let _ = match c with
+      | |- _constant _ => ()
+      | |- _atom _ => ()
+      | |- _proj _ _ => ()
+      | _ => print "only constants and atoms may have reducing information"; failure
```

Comment:
In Value.top_handle (https://github.com/Andromedans/andromeda/blob/master/src/nucleus/value.ml#L606) print the `vs`